broadcast/transaction.go

```diff
@@ -19,8 +19,14 @@ type TransactionOpts struct {
 	MerkleProof bool
 	// WaitForStatus is the status that the callback request will wait for.
 	WaitForStatus TxStatus
+	// BeefFormat is a flag which indicates that the transaction given for submitting is in BEEF Format.
+	BeefFormat bool
```

Comment:
Do not use multiple boolean flags as one switch. Please change it to enum

Reply:
Right, changed!